backend/app/views/spree/admin/shipping_methods/_form.html.erb

```diff
@@ -1,40 +1,40 @@
 <div data-hook="admin_shipping_method_form_fields" class="row">
   <div data-hook="admin_shipping_method_form_name_field" class="col-5">
     <%= f.field_container :name do %>
-      <%= f.label :name %><br />
+      <%= f.label :name, class: 'required' %>
       <%= f.text_field :name, class: 'fullwidth' %>
       <%= error_message_on :shipping_method, :name %>
     <% end %>
   </div>
 
-  <div data-hook="admin_shipping_method_form_internal_name_field" class="col-5">
+  <div data-hook="admin_shipping_method_form_admin_name_field" class="col-5">
```

Comment:
We should not change the data-hook attribute. Stores might have used Deface to override the view.